test/support/rollbackable.ex

```diff
@@ -15,7 +15,8 @@ defmodule Authable.Rollbackable do
   setup do
     # Wrap this case in a transaction
     Ecto.Adapters.SQL.Sandbox.mode(@repo, :manual)
-    :ok = Ecto.Adapters.SQL.Sandbox.checkout(@repo)
+    # :ok = Ecto.Adapters.SQL.Sandbox.checkout(@repo)
```

Comment:
what's going on here?

Reply:
There's a comment on the commit. Wasn't sure why a conn was already checked out so I didn't remove this line.

Reply:
hm, let's delete the commented out code then. after that it's good to merge.